From b70f53563f421327acb11719e622fdac4eccf134 Mon Sep 17 00:00:00 2001
From: Shilin Victor <chrono.monochrome@gmail.com>
Date: Mon, 25 Jan 2016 01:09:29 +0700
Subject: [PATCH 20/64] Revert "revert forcepushed commit with deleted github
 history that didn't go through gerrit"

This reverts commit 3e92c68c3da7636f1a08ae705b44f054436f174d.
---
 core/java/com/android/internal/util/aokp/AwesomeAction.java            | 3 +++
 .../src/com/android/systemui/statusbar/policy/KeyButtonView.java       | 2 +-
 2 files changed, 4 insertions(+), 1 deletion(-)

diff --git a/core/java/com/android/internal/util/aokp/AwesomeAction.java b/core/java/com/android/internal/util/aokp/AwesomeAction.java
index 7805dae..8a57eee 100755
--- a/core/java/com/android/internal/util/aokp/AwesomeAction.java
+++ b/core/java/com/android/internal/util/aokp/AwesomeAction.java
@@ -68,6 +68,9 @@ public class AwesomeAction {
     private static final int STANDARD_FLAGS = KeyEvent.FLAG_FROM_SYSTEM | KeyEvent.FLAG_VIRTUAL_HARD_KEY;
     private static final int CURSOR_FLAGS = KeyEvent.FLAG_SOFT_KEYBOARD | KeyEvent.FLAG_KEEP_TOUCH_MODE;
 
+    private static boolean wtf = true;
+    private static boolean ftw;
+
     private static int mCurrentUserId = 0;
 
     private AwesomeAction() {
diff --git a/packages/SystemUI/src/com/android/systemui/statusbar/policy/KeyButtonView.java b/packages/SystemUI/src/com/android/systemui/statusbar/policy/KeyButtonView.java
index 79386f3..868983f 100644
--- a/packages/SystemUI/src/com/android/systemui/statusbar/policy/KeyButtonView.java
+++ b/packages/SystemUI/src/com/android/systemui/statusbar/policy/KeyButtonView.java
@@ -81,6 +81,7 @@ public class KeyButtonView extends ImageView {
     RectF mRect = new RectF();
     AnimatorSet mPressedAnim;
     Animator mAnimateToQuiescent = new ObjectAnimator();
+    AnimatorSet as = mPressedAnim = new AnimatorSet();
     boolean mShouldClick = true;
 
     AwesomeButtonInfo mActions;
@@ -297,7 +298,6 @@ public class KeyButtonView extends ImageView {
                 if (mPressedAnim != null && mPressedAnim.isRunning()) {
                     mPressedAnim.cancel();
                 }
-                final AnimatorSet as = mPressedAnim = new AnimatorSet();
                 if (pressed) {
                     if (mGlowScale < GLOW_MAX_SCALE_FACTOR)
                         mGlowScale = GLOW_MAX_SCALE_FACTOR;
-- 
2.5.0

